1st-gen/packages/tray/README.md

```diff
@@ -87,20 +59,35 @@ This dismiss helper pattern is also implemented in the [`<sp-picker>`](https://o
 <sp-tab value="auto">Content has no buttons</sp-tab>
 <sp-tab-panel value="auto">
 
-This example shows the default behavior where the tray automatically detects that the menu content lacks dismiss buttons and renders visually hidden helpers. Screen readers will announce them as "Dismiss, button" and these helpers are keyboard accessible.
+This example shows the default behavior where the tray automatically detects that the content lacks dismiss buttons and renders visually hidden helpers. Screen readers will announce them as "Dismiss, button" and these helpers are keyboard accessible.
 
 ```html
 <overlay-trigger type="modal">
     <sp-button slot="trigger" variant="secondary">
-        Toggle menu content
+        Toggle tray content
     </sp-button>
     <sp-tray slot="click-content">
-        <sp-menu style="width: 100%">
-            <sp-menu-item>Deselect</sp-menu-item>
-            <sp-menu-item>Select Inverse</sp-menu-item>
-            <sp-menu-item>Feather...</sp-menu-item>
-            <sp-menu-item>Select and Mask...</sp-menu-item>
-        </sp-menu>
+        <div style="display: flex; flex-direction: column; margin: 16px;">
+            <p style="margin-block-start: 0;">
+                Custom content that doesn't have dismiss functionality, so the
+                tray detects it needs the visually-hidden dismiss buttons.
```

Comment:
yessss kween to incorporating contextual documentation in the example!! we should flag this as a good practice to continue!